src/energyplus/ForwardTranslator/ForwardTranslateEnergyManagementSystemActuator.cpp

```diff
@@ -65,113 +69,203 @@ namespace openstudio {
-
-            m_idfObjects.push_back(idfObject);
-            return idfObject;
-          }
-          //ZoneName object does not exist so try and translate anyway
-
-          // should not also have a space assigned
-          OS_ASSERT(!space);
-
-          // the load references a space type, this means it will reference a zonelist in EnergyPlus
-          // in EnergyPlus, each load in a zonelist is duplicated into each zone with a new name based on the zone
-          boost::optional<IdfObject> result;
-          for (const auto& space : spaceType->spaces()) {
-            auto tz = space.thermalZone();
-            if (tz) {
-              std::string tz_name = tz.get().nameString() + " " + m.nameString();
-              idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentUniqueName, tz_name);
-              idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentType, modelObject.actuatedComponentType());
-              idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentControlType, modelObject.actuatedComponentControlType());
-              if (!result) {
-                result = idfObject;
-              }
-            } else {
-              LOG(Error, "Actuator '" << modelObject.nameString() << "' references a SpaceLoad '" << load.get().name().get()
-                                      << "' which is not associated with a ThermalZone, it will not be translated.");
-            }
-          }
-          //Give WArning that spaceType has multiple spaces
-          if (m_forwardTranslatorOptions.excludeSpaceTranslation() && spaceType->spaces().size() > 1) {
-            LOG(Warn, "Actuator '" << modelObject.nameString() << "' references a SpaceLoad '" << load.get().name().get()
-                                   << "' attached to SpaceType '" << spaceType.get().nameString()
-                                   << "', with multiple spaces. Check your EMS programs to make sure your actuators are correct.");
-          }
-          // return the first idf object
-          if (result) {
-            m_idfObjects.push_back(result.get());
-          }
-          return result;
+      std::string zoneOrSpaceName = specificSpace_->nameString();
 
-        } else if (space) {
+      // Try to be helpful in case the user assigned a Space, but the FT doesn't translate spaces
+      if (m_forwardTranslatorOptions.excludeSpaceTranslation()) {
+        LOG(Warn, "Actuator '" << modelObject.nameString() << "' references a SpaceLoad '" << load_->nameString()
+                               << "' and specified that it wanted to be attached to the Space '" << specificSpace_->nameString()
+                               << "' but you have turned off ForwardTranslation's Space Feature. Falling back to using the attached ThermalZone '"
+                               << tz_->nameString() << "'.");
+        zoneOrSpaceName = tz_->nameString();
+      }
```

Comment:
User assigned a space, but we don't translate to spaces. We can recover easily from it by grabbing the ThermalZone for that space, issue a Warn.